llvm/test/CodeGen/PowerPC/aix-xcoff-reloc.ll

```diff
@@ -159,14 +159,26 @@ declare i32 @bar(i32)
 ; SYM:      Symbols [
 ; SYM-NEXT:   Symbol {
 ; SYM-NEXT:     Index: 0
-; SYM-NEXT:     Name: <stdin>
+; SYM-NEXT:     Name: .file
 ; SYM-NEXT:     Value (SymbolTableIndex): 0x0
 ; SYM-NEXT:     Section: N_DEBUG
 ; SYM-NEXT:     Source Language ID: TB_CPLUSPLUS (0x9)
 ; SYM32-NEXT:   CPU Version ID: TCPU_COM (0x3)
 ; SYM64-NEXT:   CPU Version ID: TCPU_PPC64 (0x2)
 ; SYM-NEXT:     StorageClass: C_FILE (0x67)
-; SYM-NEXT:     NumberOfAuxEntries: 0
+; SYM-NEXT:     NumberOfAuxEntries: 2
+; SYM-NEXT:     File Auxiliary Entry {
+; SYM-NEXT:       Index: 1
+; SYM-NEXT:       Name: <stdin>
+; SYM-NEXT:       Type: XFT_FN (0x0)
+; SYM64-NEXT:     Auxiliary Type: AUX_FILE (0xFC)
+; SYM-NEXT:     }
+; SYM-NEXT:     File Auxiliary Entry {
+; SYM-NEXT:       Index: 2
+; SYM-NEXT:       Name: LLVM version 18.0.0git
```

Comment:
This should not be here.

Reply:
Probably "version 18.0.0git" should be removed.